### DISCUSSION TITLE
How to add One to Many field on related Collection after creation

### DISCUSSION THREAD

**Discussion by Lucas-Gruber** [directus/directus#16085] [Archive]
Hello, i have a Collection (we name it for now collection A) and a Collection (B) and i have a Many to One Field on Collection A and Collection B is the related Collection. But i don't have on creation activated "create field on related Collection" but i don't want to type all relations again. Is there a Way to add this Field after creation?

**Comment by connorwinston** [comment]
You should just be able to make a One to Many on Collection B and select the field you created on Collection A, just tested and this works!

**Reply by Lucas-Gruber** [reply]
That helped me, thank you!
